modules/govuk/manifests/apps/bouncer.pp

```diff
@@ -173,6 +173,15 @@
     document_root => '/var/apps/bouncer/assets-directgov/directgov_campaigns',
   }
 
+  # disable X-Ray sampling, as bouncer gets a lot of traffic but
```

Comment:
I haven't looked in to X-Ray, but does this mean that we're still using X-Ray for Bouncer (it's just unsampled, rather than sampled data), or that it's being disabled?

Reply:
It'll be enabled but unsampled, as there's nothing to actually remove it in https://github.com/alphagov/govuk_app_config/blob/master/lib/govuk_app_config/govuk_xray.rb

Reply:
So I'm still uncertain around the intent of this change. Does "unsampled" mean in the context of XRay that we won't be getting any data? I'd usually interpret "unsampled" as you get all the data, rather than a sample of it.

Reply:
Ah, that was me misusing terminology.  Nothing will be sampled with a sampling rate of 0:

https://github.com/aws/aws-xray-sdk-ruby/blob/1869ca5b72dfb206e42ef92d40a7fd2f4b53f465/lib/aws-xray-sdk/sampling/local/sampler.rb#L91

Reply:
We might still get some results if I read that correctly since `Random.rand` could return `0`. Not sure how important that is though.

Reply:
I wonder if a better solution would be to update `govuk_app_config` to allow complete disabling of X-Ray (if we're setting the sample rate to 0, we may as well not bother doing any X-Ray related processing at all).

Reply:
Yeah, that is better, as with a rate of 0 there would still be a bit of overhead on each request as it decides whether to sample it or not.  I'll add a new env var to govuk_app_config to switch off -Ray entirely.

Reply:
> I'll add a new env var to govuk_app_config to switch off X-Ray entirely.

That sounds good :+1: